src/oasis-nexus/api.ts

```diff
@@ -110,17 +111,20 @@ declare module './generated/api' {
 
 export const isAccountEmpty = (account: RuntimeAccount | Account) => {
   if (account.layer === Layer.consensus) {
-    const { available, size, nonce, debonding_delegations_balance, delegations_balance, escrow, total } =
-      account as Account
-    return (
-      available === '0' &&
-      debonding_delegations_balance === '0' &&
-      delegations_balance === '0' &&
-      escrow === '0' &&
-      total === '0' &&
-      nonce === 0 &&
-      size === 'XXS'
-    )
+    // TODO: find a sane way to recognize an important consensus account.
+    // The heuristics below is clearly insufficient, because it would indicate even named accounts like "Governance Escrow" to be empty.
```

Comment:
+1

Reply:
There is already an ongoing with the Nexus team to acquire the missing data. Will follow up on this when appropriate.